tests/integration/conftest.py

```diff
@@ -0,0 +1,185 @@
+    random.seed(0)
+    num_rows = 100_000
+    data = pd.DataFrame(
+        {
+            "housing_type": [random.choice(HOUSING_TYPES) for _ in range(num_rows)],
+            "age": [str(random.random() * 100) for _ in range(num_rows)],
+            "year": [random.choice(["2020", "2030"]) for _ in range(num_rows)],
+            "race_ethnicity": [random.choice(RACE_ETHNICITIES) for _ in range(num_rows)],
+            "guardian_1": [
+                f"100_{random.randint(1,int(num_rows/3))}" for _ in range(num_rows)
+            ],
+            "first_name": [
+                "First" + "".join(random.choice(ascii_lowercase) for _ in range(3))
+                for _ in range(num_rows)
+            ],
+            "street_name": [
+                "Street" + "".join(random.choice(ascii_lowercase) for _ in range(3))
+                for _ in range(num_rows)
+            ],
+            "relation_to_household_head": [
+                random.choice(RELATIONS_TO_HOUSEHOLD_HEAD) for _ in range(num_rows)
+            ],
+            "zipcode": [str(float(random.randint(1, 99999))) for _ in range(num_rows)],
+            "date_of_birth": [
+                time.strftime(
+                    "%Y-%m-%d",
+                    time.localtime(
+                        DOB_START_DATE + random.random() * (DOB_END_DATE - DOB_START_DATE)
+                    ),
+                )
+                for _ in range(num_rows)
+            ],
+            "simulant_id": ["100_" + str(i) for i in range(num_rows)],
+            "middle_initial": [random.choice(ascii_uppercase) for _ in range(num_rows)],
+            "city": [
+                "City" + "".join(random.choice(ascii_lowercase) for _ in range(3))
+                for _ in range(num_rows)
+            ],
+            "street_number": [str(random.randint(1, 15000)) for _ in range(num_rows)],
+            "last_name": [
+                "Last" + "".join(random.choice(ascii_lowercase) for _ in range(3))
+                for _ in range(num_rows)
+            ],
+            "state": [random.choice(STATES) for _ in range(num_rows)],
+            "sex": [random.choice(["Female", "Male"]) for _ in range(num_rows)],
+            "unit_number": [
+                "Unit " + "".join(random.choice(ascii_lowercase) for _ in range(3))
+                for _ in range(num_rows)
+            ],
+            "guardian_2": [
+                f"100_{random.randint(1,int(num_rows)/4)}" for _ in range(num_rows)
+            ],
+        }
+    )
+
+    data_path = tmp_path_factory.getbasetemp() / "dummy_data.csv"
+    data.to_csv(data_path, index=False)
+
+    return data_path
+
+
+@pytest.fixture(scope="module")
+def dummy_config(tmp_path_factory):
+    """This simply copies the default config file to a temp directory
+    to be used as a user-provided config file in integration tests
+    """
+    config = get_configuration().to_dict()  # gets default config
```

Comment:
We agreed in slack convo that it's acceptable to have the user config be identical to the default config for integration test purposes.